packages/state/recoil/selectors/chain.ts

```diff
@@ -342,15 +343,30 @@ export const nativeBalancesSelector = selectorFamily<
       const balances = [
         ...get(justNativeBalancesSelector({ address, chainId })),
       ]
-      // Add native denom if not present.
       const nativeToken = getNativeTokenForChainId(chainId)
-      if (!balances.some(({ denom }) => denom === nativeToken.denomOrAddress)) {
+      const stakedDenoms = get(
+        walletTokenStakedDenomsSelector({ walletAddress: address, chainId })
+      )
```

Comment:
```suggestion
      const stakedDenoms = get(
        noWait(
          walletTokenStakedDenomsSelector({ walletAddress: address, chainId })
        )
      ).valueMaybe() || []
```

this query might take extra time depending on the number of DAOs the user interacts with, so let's make it non-blocking! `noWait` is a handy utility recoil function to do this :)

`noWait` turns it into a loadable, and then `valueMaybe` returns the value once it's loaded, and undefined otherwise. when not loaded, this just defaults to an empty array so the rest of the logic can remain the same